examples/2.x/volume.yaml

```diff
@@ -0,0 +1,43 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/4.2/types.yaml
+  - compat.yaml
+
+inputs:
+
+  keystone_username:
```

Comment:
for secrets I want to start using `openstack_auth_url` `openstack_username` etc instead of `keystone_` since this is a bit more general.

Reply:
sounds good, sure will update that